bridge-token-factory/src/lib.rs

```diff
@@ -372,6 +670,41 @@ impl BridgeTokenFactory {
             target, message
         ));
 
+        let deposit_fee_bound = self.get_deposit_token_fee_bound(&token);
+        let deposit_fee_percentage =
+            self.get_deposit_token_fee_percentage(&token)
+                .unwrap_or(DepositFeePercentage {
+                    eth_to_aurora: 0,
+                    eth_to_near: 0,
+                });
+
+        let amount_to_transfer: u128;
+        let mut fee_amount: u128;
+
+        // check if some msg attached than this call is for eth -> aurora otherwise eth -> near
+        match message.clone() {
+            Some(_message) => {
```

Comment:
```suggestion
        match message {
            Some(_) => {
```
Extra clone

Reply:
fixed in 6a7df359a29f6105fae2cccc36ad5277fb747102